WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -87,27 +87,32 @@
       <DockPanel>
         <Button Classes="plain"
                 Command="{Binding AdjustFeeCommand}"
-                IsVisible="{Binding AdjustFeeAvailable}"
                 ToolTip.Tip="Change transaction fee or confirmation time"
                 DockPanel.Dock="Right"
                 Margin="10 0">
+          <Button.IsVisible>
+            <MultiBinding Converter="{x:Static BoolConverters.Or}">
+              <Binding Path="AdjustFeeAvailable" />
+              <Binding Path="IsCustomFeeUsed" />
+            </MultiBinding>
+          </Button.IsVisible>
           <Viewbox Height="20">
             <PathIcon Data="{StaticResource adjust_fee_regular}" Foreground="{DynamicResource SystemAccentColor}" />
           </Viewbox>
         </Button>
 
         <StackPanel Spacing="10">
           <c:PreviewItem Icon="{StaticResource timer_regular}"
-                         Text="miners will work hard to confirm your transaction within">
+                         Text="miners will work hard to confirm your transaction within"
+                         IsVisible="{Binding !IsCustomFeeUsed}">
```

Comment:
Why do we hide this is custom fee rate is used?

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/6617#issuecomment-958759440